roles/confluent.kafka_broker/templates/kafka_jolokia.properties.j2

```diff
@@ -0,0 +1,13 @@
+port={{kafka_broker_jolokia_port}}
+host=0.0.0.0
+{% if kafka_broker_jolokia_ssl_enabled|bool %}
+keystore={{kafka_broker_keystore_path}}
+keystorePassword={{kafka_broker_keystore_storepass}}
+protocol=https
+sslProtocol=TLSv1.2
```

Comment:
I would suggest that we should make this a variable, for the ssl protocol, as TLS 1.3 is available, and some customers may want to move to this sooner than later.  Eventually I am sure it will become the standard as well, so it would just be a simple var update for us.

Reply:
Ya... this is a slippery slope. There are like 30 properties that could be customizable. I have this tls protocol hardcoded because a customer wants to make sure its tls1.2. But looking forward we have to decide how much support are we going to give for jolokia configs... I think we should just remove the agent entirely

Reply:
Agreed- removal is a later decision. Turning off by default would make more sense :)

The problem here is more of a 1-1 mapping of our supported variables to jolokia properties. The more variables we expose the more we have to test.
Also should we offer a "custom jolokia properties" feature? Should all props be customizable? etc... its a whole burden of logic for a component customers don't really seem to want.

But ya if you feel strongly that this should be a var, then thats easy enough to add